posthog/queries/retention/retention_events_query.py

```diff
@@ -114,7 +114,12 @@ def get_query(self) -> Tuple[str, Dict[str, Any]]:
                     ] as breakdown_values
                     """
                 ]
-            self.params.update({"start_date": self._filter.date_from, "period": self._filter.period})
+            self.params.update(
+                {
+                    "start_date": self._filter.date_from.strftime("%Y-%m-%d %H:%M:%S"),
```

Comment:
Found this one tiny bug here 🤯 

Due to not formatting like it is done further below, we keep the timezone on, leading to `07:00` time in the query instead of `00:00`.